src/test/java/org/broadinstitute/hellbender/tools/SplitNCigarReadsIntegrationTest.java

```diff
@@ -53,4 +53,15 @@ public void testSplitsFixNDN() throws Exception {
                 Arrays.asList(getTestDataDir() +"/" + "expected.splitNCigarReadsSnippet.splitNcigarReads.fixNDN.bam"));
         spec.executeTest("test fix NDN", this);
     }
+
+    /**
+     * Currently disabled due to issue: https://github.com/broadinstitute/gatk/issues/1865
+     */
+    @Test(enabled = false) //regression test for https://github.com/broadinstitute/gatk/pull/1853
+    public void testSplitsOfUnpairedAndUnmappedReads() throws Exception {
+        IntegrationTestSpec spec = new IntegrationTestSpec(
+                "-R" + b37_reference_20_21 + " -I " + largeFileTestDir + "K-562.duplicateMarked.chr20.bam -O %s",
```

Comment:
@droazen You can check in the github interface by clicking view on them if you're interested.
@meganshand Shouldn't there be a committed update to src/test/resources/large/.gitattributes to account for the new large files?